templates/artist/artist_detail.html

```diff
@@ -20,15 +20,15 @@
 {% block content %}
     <h1 class="header-font">{{ artist.name }}</h1>
     <ul class="tabs" data-tabs id="artist-detail-tabs">
-        <li class="tabs-title {% if not form.errors %}is-active{% endif %}"><a href="#overview" aria-selected="true">Overview</a></li>
+        <li class="tabs-title tab-formatting {% if not form.errors %}is-active{% endif %}"><a href="#overview" aria-selected="true">Overview</a></li>
```

Comment:
It will only overwrite the Foundation CSS where the CSS contradicts each other, which is probably what you want anyway.

Let's say I have a class called `foo` with `font-size: 18pt;`.
If later you define `foo` with `font-size: 24pt;`, it will overwrite the `foo` class and make the font bigger.
If instead you define `foo` with `background-color: #f00;`, it will keep the 18pt font size, but also set the background colour to red.